summary/summary.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+from lightning import Plugin, Millisatoshi
+import json
+import requests
+import threading
+import time
+
+plugin = Plugin(autopatch=True)
+
+
+class PriceThread(threading.Thread):
+    def run(self):
+        try:
+            r = requests.get('https://apiv2.bitcoinaverage.com/convert/global'
+                             '?from=BTC&to={}&amount=1'.format(plugin.currency))
+            plugin.fiat_per_btc = json.loads(r.content)['price']
```

Comment:
Since you're wrapping the thread in its own class, it might be nice to have `plugin` as an instance field, instead of referring to a global. Alternatively you can forgo the class and run a function in a thread using

```python
t = Thread(target=my_function, args=[plugin])
t.daemon = True  # Make sure to terminate thread if main-thread exits
t.start()
```

Reply:
Stackoverflow said make it its own class if it's refreshing the same timer, so I copied.  WIll make it daemon though.